dist/circleci/.circleci/RoboFile.php

```diff
@@ -139,6 +156,26 @@ protected function runBehatTests()
         return $tasks;
     }
 
+    /**
+     * Runs Cypress tests.
+     *
+     * @return \Robo\Task\Base\Exec[]
+     *   An array of tasks.
+     */
+    protected function runCypressTests()
+    {
+        $force = TRUE;
+        $tasks = [];
+        $tasks[] = $this->taskExec('service apache2 start');
+        $tasks[] = $this->taskFilesystemStack()
+            ->copy('.cypress/cypress.json', 'cypress.json', $force)
+            ->copy('.cypress/package.json', 'package.json', $force);
+        $tasks[] = $this->taskExec('sleep 30s');
```

Comment:
Why the sleep? Just curious.

Reply:
With behat, for some of the platforms, as starting the server was run in parallel as the behat task, that sleep made sure that the server was up and running. 

With cypress, we might or might not have the same situation. Probably this is just copy/paste as I based it on the behat set up, but if might not be needed on all platforms.